Tune_Cases/IEA15MW_robust.yaml

```diff
@@ -0,0 +1,71 @@
+  v_min:              3.                    # Cut-in wind speed [m/s] 
+  v_rated:            10.74                 # Rated wind speed [m/s]
+  v_max:              25.0                  # Cut-out wind speed [m/s], -- Does not need to be exact (JUST ASSUME FOR NOW)
+  max_pitch_rate:     0.0349                # Maximum blade pitch rate [rad/s]
+  max_torque_rate:    4500000.              # Maximum torque rate [Nm/s], {~1/4 VS_RtTq/s}
+  rated_power:        15000000.             # Rated Power [W]
+  bld_edgewise_freq:  4.0324                # Blade edgewise first natural frequency [rad/s]
+  bld_flapwise_freq:  3.4872                # Blade flapwise first natural frequency [rad/s]
+  TSR_operational:    9.0
+
+#------------------------------- CONTROLLER PARAMETERS ----------------------------------
+controller_params:
+  # Controller flags 
+  LoggingLevel:       1                     # {0: write no debug files, 1: write standard output .dbg-file, 2: write standard output .dbg-file and complete avrSWAP-array .dbg2-file
+  F_LPFType:          2                     # {1: first-order low-pass filter, 2: second-order low-pass filter}, [rad/s] (currently filters generator speed and pitch control signals)
+  F_NotchType:        0                     # Notch on the measured generator speed {0: disable, 1: enable} 
+  IPC_ControlMode:    0                     # Turn Individual Pitch Control (IPC) for fatigue load reductions (pitch contribution) {0: off, 1: 1P reductions, 2: 1P+2P reductions}
+  VS_ControlMode:     2                     # Generator torque control mode in above rated conditions {0: constant torque, 1: constant power, 2: TSR tracking PI control}
+  PC_ControlMode:     1                     # Blade pitch control mode {0: No pitch, fix to fine pitch, 1: active PI blade pitch control}
+  Y_ControlMode:      0                     # Yaw control mode {0: no yaw control, 1: yaw rate control, 2: yaw-by-IPC}
+  SS_Mode:            1                     # Setpoint Smoother mode {0: no setpoint smoothing, 1: introduce setpoint smoothing}
+  WE_Mode:            2                     # Wind speed estimator mode {0: One-second low pass filtered hub height wind speed, 1: Immersion and Invariance Estimator (Ortega et al.)}
+  PS_Mode:            3                     # Pitch saturation mode {0: no pitch saturation, 1: peak shaving, 2: Cp-maximizing pitch saturation, 3: peak shaving and Cp-maximizing pitch saturation}
+  SD_Mode:            0                     # Shutdown mode {0: no shutdown procedure, 1: pitch to max pitch at shutdown}
+  Fl_Mode:            1                     # Floating specific feedback mode {0: no nacelle velocity feedback, 1: nacelle velocity feedback}
+  Flp_Mode:           0                     # Flap control mode {0: no flap control, 1: steady state flap angle, 2: Proportional flap control}
+  # Controller parameters       
+  U_pc:               [12,14, 15, 18, 20]
```

Comment:
If I did what I was trying to do, based on the checks [here](https://github.com/nikhar-abbas/ROSCO/blob/1e84d66f431a3e26e63bd9d46b950525027ce83c/ROSCO_toolbox/controller.py#L127), and the (non) interpolation [here](https://github.com/nikhar-abbas/ROSCO/blob/1e84d66f431a3e26e63bd9d46b950525027ce83c/ROSCO_toolbox/controller.py#L252) you have two options:
1. len(U_pc) > 1, len(omega_pc) = len(zeta_pc) = 1
2. len(U_pc) = len(omega_pc) = len(zeta_pc)

I _think_ this covers standard uses cases. This all started because I wanted to allow for len(U_pc)>1 without specifying omega and zeta for the robust tuning methods. 

I verified this by changing IEA15MW_MultiOmega.yaml to have
```
  U_pc:   [14, 20]
  zeta_pc:  [2.0]             
  omega_pc:  [0.15] 
```
and no problems were caused in example_06.py.